conf/modules.config

```diff
@@ -166,7 +166,7 @@ process {
         ]
     }
     withName: 'ENSEMBLVEP_VEP' {
-        //ext.args         ='--everything --filter_common --per_gene --total_length --offline'
+        ext.args         ='--everything --filter_common --per_gene --total_length --offline'
```

Comment:
Please exclude `--filter_common` and `--per_gene`, as we need all annotations for all variants, regardless of MAF. We also require all consensus data, not just the 'most severe' consensus for each variant.

Reply:
these parameters are mainly default settings, update on those parameters has to be given by local config. But I will this as well for now.